### ISSUE TITLE
WARNING: HighSeas does not connect to: map:247 in europe:20

### ISSUE THREAD

**Issue by Beep6581** [FreeCol/freecol#44]
Trying to sail my ship from Europe to New Holland

`nightly-2018-08-22-403-g`ac4d931b6

[5d020af_Dutch_1512.fsg.zip](https://github.com/FreeCol/freecol/files/3366141/5d020af_Dutch_1512.fsg.zip)

```
FreeCol game version: 0.11.6 (Revision: ac4d931b6)
FreeCol protocol version: 0.1.6

Java vendor: Oracle Corporation
Java version: 1.8.0_191
Java WM name: OpenJDK 64-Bit Server VM
Java WM vendor: Oracle Corporation
Java WM version: 25.191-b12

OS name: Linux
OS architecture: amd64
OS version: 5.0.0-sabayon

net.sf.freecol.common.util.LogBuilder log
	INFO: sun.java2d.pmoffscreen using client option: true, sun.java2d.opengl overrides client option: true, sun.java2d.xrender using client option: true
	Mon Jul 08 00:03:11 CEST 2019
	Thread: 1
net.sf.freecol.FreeCol main
	INFO: Configuration:
	  version     0.11.6 (Revision: ac4d931b6)
	  java:       1.8.0_191
	  memory:     954728448
	  locale:     en_SE
	  data:       /home/morgan/programs/code-freecol/data
	  userConfig: /home/morgan/.config/freecol
	  userData:   /home/morgan/.local/share/freecol
	  autosave:   /home/morgan/.local/share/freecol/save/autosave
	  logFile:    /home/morgan/.cache/freecol/FreeCol.log
	  options:    /home/morgan/.config/freecol/freecol/options.xml
	  save:       /home/morgan/.local/share/freecol/save
	  userMods:   /home/morgan/.local/share/freecol/mods
	  debug:      
	Mon Jul 08 00:03:11 CEST 2019
	Thread: 1
net.sf.freecol.client.gui.SwingGUI <init>
	INFO: GUI constructed using scale factor 1.25
	Mon Jul 08 00:03:11 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.FontResource <init>
	INFO: Loaded font: ShadowedBlack from: file:/home/morgan/programs/code-freecol/data/base/resources/fonts/ShadowedBlack.ttf
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.FontResource <init>
	INFO: Loaded font: Dialog.plain from: urn:font:Dialog-PLAIN-12
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.FontResource <init>
	INFO: Loaded font: Serif.plain from: urn:font:Serif-PLAIN-12
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.ResourceManager addMapping
	INFO: Resource manager adding mapping base
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread started
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 15
net.sf.freecol.common.resources.ResourceManager addMapping
	INFO: Resource manager adding mapping tc
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread started
	Mon Jul 08 00:03:12 CEST 2019
	Thread: 17
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread cancelled after it preloaded 38 resources.
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 15
net.sf.freecol.client.FreeColClient loadClientOptions
	INFO: Load default client options.
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.common.option.OptionGroup load
	INFO: Load OptionGroup(clientOptions) from /home/morgan/programs/code-freecol/data/base/client-options.xml succeeded
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.client.FreeColClient loadClientOptions
	INFO: Load client options from the action manager.
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.client.FreeColClient loadClientOptions
	INFO: Merge client options from user options file: /home/morgan/.config/freecol/freecol/options.xml
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.common.option.OptionGroup load
	INFO: Load OptionGroup(clientOptions) from /home/morgan/.config/freecol/freecol/options.xml succeeded
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.client.control.SoundController <init>
	INFO: Create sound controller with net.sf.freecol.common.option.AudioMixerOption:model.option.audioMixer/[model.option.audioVolume 100%]
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.client.gui.Canvas <init>
	INFO: Windowed mode used.
	Mon Jul 08 00:03:13 CEST 2019
	Thread: 1
net.sf.freecol.client.gui.FreeColFrame <init>
	INFO: Frame created in size 1920x1050
	Mon Jul 08 00:03:14 CEST 2019
	Thread: 1
net.sf.freecol.client.gui.FreeColFrame$1 componentResized
	INFO: Window size changes to java.awt.Dimension[width=1920,height=1054]
	Mon Jul 08 00:03:14 CEST 2019
	Thread: 21
net.sf.freecol.client.gui.Canvas <init>
	INFO: Canvas created.
	Mon Jul 08 00:03:14 CEST 2019
	Thread: 1
net.sf.freecol.client.gui.SwingGUI startGUI
	INFO: GUI started.
	Mon Jul 08 00:03:14 CEST 2019
	Thread: 1
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread preloaded 1435 resources.
	Mon Jul 08 00:03:15 CEST 2019
	Thread: 17
net.sf.freecol.client.gui.FreeColFrame$1 componentResized
	INFO: Window size changes to java.awt.Dimension[width=1920,height=1054]
	Mon Jul 08 00:03:15 CEST 2019
	Thread: 21
net.sf.freecol.client.gui.FreeColFrame$1 componentResized
	INFO: Window size changes to java.awt.Dimension[width=1920,height=1054]
	Mon Jul 08 00:03:15 CEST 2019
	Thread: 21
net.sf.freecol.common.sound.SoundPlayer openLine
	WARNING: Mixer does not support interface SourceDataLine supporting format PCM_SIGNED 48000.0 Hz, 16 bit, stereo, 4 bytes/frame, little-endian
	Mon Jul 08 00:03:16 CEST 2019
	Thread: 20
net.sf.freecol.client.ClientOptions load
	INFO: Load OptionGroup(clientOptions) from /home/morgan/.local/share/freecol/save/5d020af_Dutch_1512.fsg succeeded
	Mon Jul 08 00:03:19 CEST 2019
	Thread: 21
net.sf.freecol.FreeCol setName
	INFO: Set FreeCol.name = Morgan
	Mon Jul 08 00:03:19 CEST 2019
	Thread: 21
net.sf.freecol.server.FreeColServer createServer
	WARNING: Could not resolve local host name
	Mon Jul 08 00:03:20 CEST 2019
	Thread: 21
net.sf.freecol.server.FreeColServer readGame
	INFO: Found savegame version 14
	Mon Jul 08 00:03:20 CEST 2019
	Thread: 21
net.sf.freecol.common.model.Specification clean
	INFO: Specification clean following load from stream complete, starting year=1492, season year=1600, ages=[1,109,309], seasons=2, difficulty=model.difficulty.medium, 267 Types, 80 Abilities, 41 BuildingTypes, 14 Disasters, 10 EuropeanNationTypes, 2 Events, 25 FoundingFathers, 21 GoodsTypes, 8 IndianNationTypes, 59 Modifiers, 25 Nations, 166 Options, 23 Option Groups, 12 ResourceTypes, 11 Roles, 23 TileTypes, 6 TileImprovementTypes, 13 UnitChangeTypes, 42 UnitTypes read.
	Mon Jul 08 00:03:20 CEST 2019
	Thread: 21
net.sf.freecol.server.FreeColServer loadGame
	INFO: Game integrity test failed, but fixed.
	  Removed broken river connection to NE at 21,9-borealForest
	  Removed broken river connection to SE at 24,100-rainForest
	  Removed broken river connection to SE at 33,119-coniferForest
	  Removed broken river connection to NE at 31,138-mixedForest
	  Removed broken river connection to SW at 37,152-rainForest
	Mon Jul 08 00:03:20 CEST 2019
	Thread: 21
net.sf.freecol.server.FreeColServer loadGame
	INFO: AI integrity test succeeded.
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.common.resources.ResourceManager addMapping
	INFO: Resource manager adding mapping game
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread started
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 40
net.sf.freecol.common.resources.ResourceManager$1 run
	INFO: Preload background thread preloaded 1435 resources.
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 40
net.sf.freecol.client.control.ConnectController connect
	INFO: Connected to 127.0.0.1:3541 as Morgan
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.client.control.ConnectController requestLogin
	INFO: Login request for client Morgan
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.server.FreeColServer addNewUserConnection
	INFO: Client connected from /127.0.0.1:57220
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 39
net.sf.freecol.common.model.Specification clean
	INFO: Specification clean following load from stream complete, starting year=1492, season year=1600, ages=[1,109,309], seasons=2, difficulty=model.difficulty.medium, 267 Types, 80 Abilities, 41 BuildingTypes, 14 Disasters, 10 EuropeanNationTypes, 2 Events, 25 FoundingFathers, 21 GoodsTypes, 8 IndianNationTypes, 59 Modifiers, 25 Nations, 166 Options, 23 Option Groups, 12 ResourceTypes, 11 Roles, 23 TileTypes, 6 TileImprovementTypes, 13 UnitChangeTypes, 42 UnitTypes read.
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 42
net.sf.freecol.client.control.ConnectController login
	INFO: Login accepted for client Morgan to ready single-player game as Morgan/player:19
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.server.control.PreGameController requestLaunch
	INFO: Launching for [ServerPlayer player:19 Morgan]
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 45
net.sf.freecol.server.FreeColServer startGame
	INFO: Server starting game: LOAD_GAME
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 45
net.sf.freecol.client.control.PreGameController$1 run
	INFO: Client starting game
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 47
net.sf.freecol.common.sound.SoundPlayer openLine
	WARNING: Mixer does not support interface SourceDataLine supporting format PCM_SIGNED 48000.0 Hz, 16 bit, stereo, 4 bytes/frame, little-endian
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 20
net.sf.freecol.client.gui.panel.InfoPanel update
	INFO: InfoPanel NONE -> ENDinconsistent
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.client.gui.SwingGUI enableMapControls
	INFO: Instantiated net.sf.freecol.client.gui.panel.CornerMapControls
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.client.gui.panel.InfoPanel$UnitInfoPanel update
	INFO: Switching UnitInfoPanel from null to unit:21970 Vogel Struys (Merchantman) 5/5
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.client.gui.panel.InfoPanel update
	INFO: InfoPanel END -> UNIT
	Mon Jul 08 00:03:21 CEST 2019
	Thread: 21
net.sf.freecol.common.model.Player clientError
	WARNING: HighSeas does not connect to: map:247 in europe:20
	Mon Jul 08 00:03:46 CEST 2019
	Thread: 49
```

**Comment by wintertime** [comment]
Please, check if you can sail a ship to Europe and back now with newest nightly version! (If it is already in Europe it may be necessary to use an older save and sail to Europe again.)

**Comment by Beep6581** [comment]
`nightly-2018-08-22-479-g`8778bf4eb

Using the saved game from the first post the game still crashes when I tried sailing the ship which is already in Europe back to the new world.

However, when starting a new game (building a colony, moving the ship into it, then sailing the ship to Europe), sailing the ship from Europe back to the new world works correctly.

**Comment by wintertime** [comment]
Thank you. So it's as I thought. You should be able to use an old savegame where the ship is not in Europe, though.

**Comment by mpope042** [comment]
The saved game works for me now, and I am pretty sure we fixed this in BR#3167 (git.58660f4a) back in January.
